### ISSUE TITLE
COR (Complex Object Retrieval) Prefetching support

### ISSUE THREAD

**Issue by wwanderley** [oracle/odpi#123]
Thank you for lib!

ODPI-C doesn't seem to support COR Prefetching 
https://docs.oracle.com/cd/E11882_01/appdev.112/e10646/oci11obj.htm#LNOCI16806

Do you have plans to add this feature?

**Comment by anthony-tuininga** [comment]
No, it does not support COR prefetching. It may not be relevant, however, as I am currently always using VALUE instead of REF -- so there is no need to call ``OCIObjectPin()``. Can you give some indication of why you feel this might be a good enhancement? Do you have the need for handling references to objects intead of just values?
